src/cli/commands/init.js

```diff
@@ -114,14 +121,18 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
       def = val;
     }
 
+    if (manifestKey === 'private' && privateFlag) {
+      def = true;
+    }
+
     if (def) {
-      question += ` (${def})`;
+      question += ` (${def.toString()})`;
```

Comment:
That's interesting. That is Flow complaining about the type. I didn't think it would complain inside an interpolated string. Oh well, in that case the `.toString()` is fine.